src/tests/topp/MRMTransitionGroupPicker_1_output.featureXML

```diff
@@ -372,7 +372,7 @@
 					<UserParam type="string" name="native_id" value="60349_NSFVSYENFYQNHENAEIR/3_y16_2"/>
 					<UserParam type="float" name="peak_apex_int" value="50"/>
 					<UserParam type="float" name="total_xic" value="6777"/>
-					<UserParam type="float" name="width_at_50" value="6.82999999999993"/>
+					<UserParam type="float" name="width_at_50" value="3.42000000000007"/>
```

Comment:
are these values now correct?

Reply:
the feature goes from 

```
<UserParam type="float" name="leftWidth" value="4152.9599609375"/>
<UserParam type="float" name="rightWidth" value="4166.6201171875"/>
```

which are 13 seconds, so it seems like the `width_at_50` for the other ones is simply the full width of the feature except for this one feature here. Have you looked at the feature?